wunderio/client-UK-FSA-beta
# Bug/fsa 925 news and alerts search facets

To test:
- Search for something
- All the filters on 'guidance', 'news and alerts', 'research' should look consistent.
- Note that the filters on 'food hygiene ratings' is still todo

FSA-1017 (open the top filter by default on desktop) is also related to this task, but I'll fix next week. Better to separate them, than have that feature block this bugfix.
